src/controllers/khbooks/invoice-controller.ts

```diff
@@ -363,7 +363,7 @@ export const send = async (req: Request, res: Response) => {
 export const sendReminder = async (req: Request, res: Response) => {
   try {
     const { id } = req.params
-    await InvoiceService.sendInvoice(parseInt(id))
+    await InvoiceService.sendInvoice(parseInt(id), "Reminder")
```

Comment:
Let's create an `SendInvoiceType` enum for this.

Reply:
done :3